cmd/influxd/config.go

```diff
@@ -74,15 +74,9 @@ type Config struct {
 
 	InputPlugins struct {
 		UDPInput struct {
-			Enabled  bool   `toml:"enabled"`
-			Port     int    `toml:"port"`
-			Database string `toml:"database"`
+			Enabled bool `toml:"enabled"`
```

Comment:
You should ensure the test config string in `config_test.go` is updated with the relevant changes, so we can sure it parses.